src/views/Generator/TestOptions/ThinkTime.tsx

```diff
@@ -1,21 +1,23 @@
 import { useCallback, useEffect } from 'react'
 import { useForm } from 'react-hook-form'
 import { zodResolver } from '@hookform/resolvers/zod'
-import { Box, Flex, TextField, Text } from '@radix-ui/themes'
+import { Box, Flex, TextField, Text, Grid } from '@radix-ui/themes'
 
-import { useGeneratorStore } from '@/store/generator'
+import { selectHasGroups, useGeneratorStore } from '@/store/generator'
 import { ThinkTimeSchema } from '@/schemas/testOptions'
 import type { ThinkTime } from '@/types/testOptions'
 import { stringAsNullableNumber, stringAsOptionalNumber } from '@/utils/form'
-import { ControlledSelect, FieldGroup } from '@/components/Form'
+import { FieldGroup } from '@/components/Form'
+import { ControlledRadioGroup } from '@/components/Form/ControllerRadioGroup'
 
 const TYPE_OPTIONS = [
   { value: 'fixed', label: 'Fixed' },
-  { value: 'range', label: 'Range' },
+  { value: 'range', label: 'Random' },
```

Comment:
Not sure about this one, but I though it'd be a bit clearer than `range`. Here are some other alternatives:
* Static - Dynamic
* Fixed - Random in range

Apart from that, it may be a good idea to have `range` selected by default OR mark it as "recommended"